### PR TITLE
Update mutator

### PR DESCRIPTION
**Pull request by aljazkosir**
Update sensu mutator to the new module structure

### CODE REVIEW

File: plugins/modules/mutator.py [hunk 1]
```diff
@@ -0,0 +1,115 @@
...
+# Copyright: (c) 2019, XLAB Steampunk <steampunk@xlab.si>
+#
+# GNU General Public License v3.0+ (see COPYING or https://www.gnu.org/licenses/gpl-3.0.txt)
+
+from __future__ import absolute_import, division, print_function
+__metaclass__ = type
+
+ANSIBLE_METADATA = {'metadata_version': '1.1',
+                    'status': ['preview'],
+                    'supported_by': 'community'}
```
Comment (miha-plesko) [inline plugins/modules/mutator.py]: Didn't we usually put "XLAB Steampunk" here?

File: plugins/modules/mutator.py [hunk 2]
```diff
@@ -0,0 +1,115 @@
...
+  - sensu.sensu_go.base
+  - sensu.sensu_go.object
+options:
+  command:
+    description:
+      - The mutator command to be executed by the Sensu backend.
+    type: str
+  timeout:
+    description:
+      - Timeout for mutator execution
```
Comment (miha-plesko) [inline plugins/modules/mutator.py]: Is there a unit? Seconds?

File: tests/integration/modules/molecule/mutator/molecule.yml [hunk 3]
```diff
@@ -0,0 +1,22 @@
+---
+scenario:
+  name: mutator
+platforms:
+  - name: latest
+    image: sensu/sensu:latest
+    command: >
+      sensu-backend start
+        --state-dir /var/lib/sensu/sensu-backend/etcd1
+        --log-level debug
```
Comment (miha-plesko) [inline tests/integration/modules/molecule/mutator/molecule.yml]: It just stroke me we could use YAML anchors in those files so we didn't have to repeat the command over and over again, something like:

```yaml
- name: latest
  image: sensu/sensu:latest
  command: &cmd >
    sensu-backend start
      --state-dir /var/lib/sensu/sensu-backend/etcd1
      --log-level debug
- name: v5.13.1
  image: sensu/sensu:5.13.1
  command: *cmd
- name: v5.12.0
  image: sensu/sensu:5.12.0
  command: *cmd
```

Apply at will, it's not a must have

File: tests/unit/modules/test_mutator.py [hunk 4]
```diff
@@ -0,0 +1,81 @@
...
+            ),
+        )
+        assert check_mode is False
+
+    def test_failure(self, mocker):
+        sync_mock = mocker.patch.object(utils, 'sync')
+        sync_mock.side_effect = errors.Error('Bad error')
+        set_module_args(
+            name='test_mutator',
+            command='/bion/true'
```
Comment (miha-plesko) [inline tests/unit/modules/test_mutator.py]: Ahhhh this reminds me of the LEGO Bionicle. Ever played with [those](https://www.google.com/search?q=bionicle&client=firefox-b-d&tbm=isch&source=iu&ictx=1&fir=G2dj5JWKpZVJnM%253A%252CaV4NZ2PHs6O1XM%252C_&vet=1&usg=AI4_-kSsa3GpG1it-UCh_H7R3m8Iiezp9w&sa=X&ved=2ahUKEwik6qWK6ejkAhUIExoKHZ_RCGoQ9QEwA3oECAIQDw#imgrc=LVpzbLGS6fYHUM:&vet=1)? 😂 

Please don't fix the typo here, it's so amusing (and doesn't affect the test in any way) 👍

Comment (aljazkosir) [reply tests/unit/modules/test_mutator.py]: Haha yeah I did, I still have them somewhere :laughing:

**Pull request merged**